AUTHORS

```diff
@@ -0,0 +1,5 @@
+# This is the list of ths significant contributors.
+
+Kung-Yao Hsieh
+Shang-Chen LIN
+Yi-Hang Xie
```

Comment:
Please add new line at the end.

Reply:
Remove author file and use license file only instead.